packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -202,6 +203,7 @@ describe('HCP home page', () => {
 
     // check the manual flag toggle
     await updatePatientsFilters({ ...defaultToggles, manualFlagFilterToggle: true })
+    expect(screen.getByTestId('filters-label')).toHaveTextContent('Filters activated: 1 patient(s) out of 6')
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏This check has already been done in `checkDataGridAfterSinglePatientFilter`. You can remove all the one that you added :wink: